src/main/groovy/edu/oregonstate/mist/personsapi/mapper/PersonMapper.groovy

```diff
@@ -0,0 +1,33 @@
+package edu.oregonstate.mist.personsapi.mapper
+
+import edu.oregonstate.mist.personsapi.core.PersonObject
+import edu.oregonstate.mist.personsapi.PhoneFormatter
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+
+import java.sql.ResultSet
+import java.sql.SQLException
+
+public class PersonMapper implements ResultSetMapper<PersonObject> {
+    PhoneFormatter phoneFormatter = new PhoneFormatter()
+    public PersonObject map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        new PersonObject (
+            osuID: rs.getString('OSU_ID'),
+            firstName: rs.getString('FIRST_NAME'),
+            lastName: rs.getString('LAST_NAME'),
+            middleName: rs.getString('MIDDLE_NAME'),
+            birthDate: rs.getDate('BIRTH_DATE'),
+            primaryAffiliation: rs.getString('PRIMARY_AFFILIATION'),
+            email: rs.getString('EMAIL_ADDRESS'),
+            username: rs.getString('USERNAME'),
+            osuUID: rs.getString('OSUUID'),
+            confidential: rs.getBoolean('CONFIDENTIAL'),
```

Comment:
I tried doing a query with a confidential person and it threw a 500 error. I think it's because the confidential indicator uses `Y` or `null`, not 1 or 0.